nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -4862,4 +4794,46 @@ mod tests {
             );
         }
     }
+
+    #[tokio::test]
+    async fn insert_target_query_sql_snapshot() {
```

Comment:
nit, take it or leave it: most of the other tests of this form i've seen are named `expectorate_{query}`, maybe we should make them consistent?

Reply:
Renamed